pravega/pravega-operator
# Issue 314: Go version changed to 1.13 so as to switch use to go modules

# Purpose of the change
Fixes #314

# what the code does:
previously we were not using go modules because of which we had a restriction of having the code in the go path for it to work with these changes we will be able to even run our code without it being in the go path

# How to verify it :
1) All the make commands in the make file are working as expected
2) Checkout and build code outside $GOPATH should work.
3)  All basic operator functionality - Cluster Creation, Deletion, Scaling-up & Down, Upgrades and 
    Rollback work as expected.

**Note**- I have verified all the above changes
